cmd/migrate.go

```diff
@@ -0,0 +1,33 @@
+package cmd
+
+import (
+	"context"
+
+	"github.com/spf13/cobra"
+
+	"go.infratographer.com/identity-api/internal/config"
+	"go.infratographer.com/identity-api/internal/storage"
+)
+
+func init() {
+	rootCmd.AddCommand(runMigrationsCmd)
+}
+
+var runMigrationsCmd = &cobra.Command{
+	Use:   "run-migrations",
```

Comment:
nit: We used `migrate` for fertilesoil: https://github.com/infratographer/fertilesoil/blob/main/cmd/migrate.go#L16

looks a little easier to type than this.

Reply:
I originally called it `migrate` (hence the file name) but wasn't sure if that conveyed enough information. Happy to change it back.

Reply:
I leave this decision up to you. I don't think it's a big deal

Reply:
I'd rather be consistent and `migrate` is a perfectly good name. Done!